docs/api-v1.md

```diff
@@ -577,4 +579,38 @@ Base endpoint is: `/ocs/v2.php/apps/spreed/api/v1`
         
 ## Signaling
 
-See the [Draft](https://github.com/nextcloud/spreed/wiki/Signaling-API) in the wiki…
+### Get signaling settings
+
+* Method: `GET`
+* Endpoint: `/signaling/settings`
+* Data:
+
+    field | type | Description
+    ------|------|------------
+    `stunservers` | array | STUN servers
+    `turnservers` | array | TURN servers
+    `server` | string | URL of the external signaling server
+    `ticket` | string | Ticket for the external signaling server
+
+    - STUN server
+    
+       field | type | Description
+       ------|------|------------
+       `url` | string | STUN server URL
+
+    - TURN server
+    
+       field | type | Description
+       ------|------|------------
+       `url` | array | One element array with TURN server URL
+       `urls` | array | One element array with TURN server URL
```

Comment:
url and urls?

Reply:
That's what we send at the moment. I already discussed with @fancycode some changes here.
But they will require changes in mobiles apps as well.